### PR TITLE
feat(contacts): add contacts component - INNO-368

### PR DESCRIPTION
**Pull request by kalinchernev**

### CODE REVIEW

File: src/framework/atoms/button/styles/button.less [hunk 1]
```diff
@@ -17,3 +17,7 @@
   border: 1px solid #fff;
   background: @brand-info;
 }
+
+.btn-full {
```
Comment (emeryro) [inline src/framework/atoms/button/styles/button.less]: There is a class in Bootstrap already available for this: .btn-block

File: src/framework/molecules/contacts/templates/contacts.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  </div>
+  <div class="contacts__pane user-profile-block-language">
+    <div><strong>Language</strong></div>
+    <ul>
+      <li typeof="skos:Concept" property="rdfs:label skos:prefLabel" >Bulgarian</li>
+      <li typeof="skos:Concept" property="rdfs:label skos:prefLabel" >French</li>
+      <li typeof="skos:Concept" property="rdfs:label skos:prefLabel" >English</li>
+    </ul>
+  </div>
+  <div class="contacts__pane user-profile-block-social-media">
```
Comment (emeryro) [inline src/framework/molecules/contacts/templates/contacts.html]: Maybe icons should be a little large. In theory we should have some modifiers to make text larger/smaller, but here I guess one line of css is enough for these icons.

File: src/SUMMARY.md [hunk 3]
```diff
@@ -24,7 +24,7 @@
         *   [✗ Comment](./framework/molecules/comment/README.md)
         *   [✓ Community create button](./framework/molecules/community-create-button/README.md)
         *   [✓ Community navigation](./framework/molecules/community-navigation/README.md)
-        *   [✗ Contacts](./framework/molecules/contacts/README.md)
+        *   [✓ Contacts](./framework/molecules/contacts/README.md)
```
Comment (emeryro) [inline src/SUMMARY.md]: Do not forget to change it in readme file too ;)

File: src/framework/molecules/contacts/templates/contacts.html (hunk 2 above)
Comment (kalinchernev) [inline src/framework/molecules/contacts/templates/contacts.html]: Agree for both.

**Comment by kalinchernev** [comment]
Changed the icons without too much abstraction. Added also a small hover effect, because @degliwe said once we have to make small things here and there which make us happy :) (If you don't like it, please tell me)

**Pull request merged**